docs/examples/entur/build-config.json

```diff
@@ -20,9 +20,9 @@
   "extraEdgesStopPlatformLink": true,
   "distanceBetweenElevationSamples": 25,
   "multiThreadElevationCalculations": true,
-  "streetGraph": "gs://<bucket>/${OTP_GCS_BASE_GRAPH_PATH}",
-  "graph": "gs://<bucket>/${OTP_GCS_WORK_DIR}/Graph-otp2-${otp.serialization.version.id}.obj",
```

Comment:
The configs should go trough `EnvironmentVariableReplacer` before validation, so these can be left as is

Reply:
I've used the `VariableReplacer`.